server/src/test/java/io/crate/integrationtests/DynamicMappingUpdateITest.java

```diff
@@ -23,6 +23,7 @@
 
 import static io.crate.testing.Asserts.assertThat;
 import static io.crate.testing.TestingHelpers.printedTable;
+import static org.assertj.core.api.Assertions.assertThat;
```

Comment:
not needed?

Reply:
For some reason eclipse adds it back automatically if I remove it.